aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1309,6 +1309,19 @@ Tensor outer(const Tensor& self, const Tensor& vec2) {
   return self.reshape_symint({self.sym_size(0), 1}) * vec2;
 }
 
+#ifdef __aarch64__
+static inline bool is_mkldnn_matmul_enabled() {
+  static bool value = [&](const char* pt) {
+    if (pt != nullptr) {
+      return std::atoi(pt);
+    } else {
+      return 1;
+    }
+  }(std::getenv("MKLDNN_MATMUL_ENABLE"));
```

Comment:
Nits: Please use `TORCH_` prefix for all PyTorch env vars and use ternaries to make code a bit more compact.
```suggestion
  static bool value = [&]() {
    const char *ptr = std::getenv("TORCH_MKLDNN_MATMUL_ENABLE");
     return pt != nullptr ? std::atoi(pt) : 1;
  }();
```